### ISSUE TITLE
NVDA is not giving any info to the user if any object(Picture/SmartArt/Shape etc) contains hyperlinks

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#9397]
### Steps to reproduce:
1. Open a word document 
2. Insert a picture or any other object to the doument
3. Select these objects and insert hyperlink by pressing Ctrl+K
4. Navigate to the objects using Down arrow keys line by line and observe NVDA announcement for the objects containing hyperlinks. 
5. NVDA doesn't give info to the user about hyperlink. 

### Actual behavior:
If we navigate to a Picture containing hyperlinks, NVDA is announcing as '/' slash but it doesn't give any info about hyperlinks. 

### Expected behavior:
NVDA should announce that object 'Contains hyperlinks'. 

#### NVDA installed/portable/running from source:
Installed

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
NO
